inert/dynamic-inert-on-focused-element.tentative.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Dynamic inertness on focused element</title>
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#focus-fixup-rule">
+<meta name="assert" content="If a focused element becomes expressly inert, it stops being focused.">
+<div id="log"></div>
+
+<input class="check-focus becomes-inert">
+
+<div class="becomes-inert">
+  <input class="check-focus">
+</div>
+
+<button class="check-focus becomes-inert">foo</button>
+
+<div class="check-focus becomes-inert" tabindex="-1"></div>
+
+<div class="becomes-inert">
+  <div class="check-focus" tabindex="-1">bar</div>
+</div>
+
+<div class="becomes-inert">
+  <p>
+    <span class="check-focus" tabindex="-1">baz</span>
+  </p>
+</div>
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+function nextRepaint() {
+  return new Promise(resolve => {
+    requestAnimationFrame(() => {
+      requestAnimationFrame(resolve);
+    });
+  });
+}
+
+const loaded = new Promise(resolve => {
+  addEventListener("load", resolve, {once: true});
+});
+promise_setup(() => loaded);
+
+for (const element of document.querySelectorAll(".check-focus")) {
+  const becomesInert = element.closest(".becomes-inert");
+  promise_test(async function() {
+    element.focus();
+    assert_equals(document.activeElement, element, "The element is focused");
+
+    becomesInert.inert = true;
+    await nextRepaint();
+    assert_equals(document.activeElement, document.body, "The element stops being focused");
+  }, becomesInert.outerHTML);
```

Comment:
not a big fan of this subtest name, this will probably appear wonky in WebKit test expectations.

Could you either put custom messages/element ids or just use indices (e.g. test 1, 2)?